bitcoin/src/lib.rs

```diff
@@ -204,6 +212,15 @@ impl BitcoinCore {
             network,
             transaction_creation_lock: Arc::new(Mutex::new(())),
             connection_timeout,
+            electrs_config: ElectrsConfiguration {
+                base_path: match network {
```

Comment:
Would be great if we could override this with a custom URL, then use of the the consts if unset. This means we don't have to re-build the vault client if one of these changes for whatever reason.